docs/Inference-config.yml

```diff
@@ -9,7 +9,7 @@ model_dir: '/home/Translation_temp_2/'
 # directory with images on which you want to perform the prediction, the inference dataset
 image_dir: '/home/InferenceData/'
 
-# directory with all preprocessed data
+# preprocess_dir contains the preprocessing_info.json file, to extract information about normalization step
 preprocess_dir: '/home/Processed_temp_1'
 
 # define inference dataset channels
```

Comment:
Instead of slice_id be commented out it could be defined as slice_id = -1 to show this option.